src/utils.js

```diff
@@ -1,18 +1,30 @@
-import { omit } from 'lodash';
+import { omit, identity } from 'lodash';
 
 export const parseInclude = request => {
   const include = Array.isArray(request.query.include) ? request.query.include
                                                        : [request.query.include];
 
+  const noGetDb = typeof request.getDb !== 'function';
+  const noRequestModels = request.models;
```

Comment:
It seems like `noRequestModels` is not what you want, it's truthy if `request.models` exists, maybe you want `!request.models`?